bin/sozo/src/commands/events.rs

```diff
@@ -82,8 +97,11 @@ impl EventsArgs {
                 keys,
             };
 
-            let res =
-                provider.get_events(event_filter, self.continuation_token, self.chunk_size).await?;
+            let res = provider.get_events(
+                event_filter,
+                self.continuation_token,
+                self.chunk_size
+            ).await?;
```

Comment:
This function call is the one that must take in account some block ranges, and cumulate the data for the blocks.

The data will remain in memory until all the world's data are gathered. We may start thinking how big world's data could be split, but this can be done later. For now, removing the world block could be great to avoid this additional settings since we can implement a logic solving that.